### PR TITLE
Header menu footer

### PR DESCRIPTION
**Pull request by michal-karalus**

### CODE REVIEW

File: src/components/Footer/Footer.module.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.wrapper {
+  padding: 0 24px;
+
+  a {
```
Comment (sasza2) [inline src/components/Footer/Footer.module.scss]: a little bit too generic name. maybe check something like this:
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <div className={styles.wrapper}>
+        <span>Application is using </span>
+        <a
+          href="https://spoonacular.com/food-api/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Spoonacular API
+        </a>
+        <span> to provide recipes data.</span>
```
Comment (sasza2) [inline src/components/Footer/index.tsx]: maybe you're looking for `&nbsp;` instead of space

File: src/components/Header/Header.module.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    align-items: center;
+    height: var(--header-height);
+  }
+}
+
+.logo {
+  text-decoration: none;
+  color: var(--black);
+
+  h1 {
```
Comment (sasza2) [inline src/components/Header/Header.module.scss]: same as above

File: src/components/Search/Search.module.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .container {
-  height: 100vh;
+  height: calc(100vh - var(--header-height) - var(--footer-height));
```
Comment (sasza2) [inline src/components/Search/Search.module.scss]: hmm, cant be simple like 100% ?

File: src/components/SideMenu/index.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+import { Link } from 'react-router-dom'
+import classnames from 'classnames'
+
+import styles from './SideMenu.module.scss'
+
+function SideMenu() {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const toggleMenu = () => {
+    setIsOpen((current) => !current)
```
Comment (sasza2) [inline src/components/SideMenu/index.tsx]: setIsOpen(!current) should work here, its not in any useEffect / useCallback etc

File: src/components/SideMenu/index.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  return (
+    <>
+      <button
+        className={classnames(
+          styles.hamburger,
+          isOpen ? styles.hamburgerActive : ''
```
Comment (sasza2) [inline src/components/SideMenu/index.tsx]: isOpen && styles.hamburgerActive wont work for classnames? does it omit all falsy values?

File: src/components/Search/Search.module.scss (hunk 4 above)
Comment (michal-karalus) [reply src/components/Search/Search.module.scss]: It didn't work. I've decided to use solution described here https://css-tricks.com/quick-css-trick-how-to-center-an-object-exactly-in-the-center/
I've also done a small change to form's styles. This form will probably be rewritten in the future anyway coz I'll extend it with more inputs

File: src/components/Search/Search.module.scss [hunk 7]
```diff
@@ -1,8 +1,8 @@
 .container {
-  height: 100vh;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
Comment (sasza2) [inline src/components/Search/Search.module.scss]: hmm, I dont get it. what's the point?

Comment (michal-karalus) [reply src/components/Search/Search.module.scss]: Okey, I've centered it with flexbox and had to do few other changes

**Review by sasza2 — APPROVED** [review]

**Pull request merged**